### ISSUE TITLE
Update provider icon

### ISSUE THREAD

**Issue by jaywcarman** [ManageIQ/manageiq-providers-ibm_power_vc#10]
The [existing icon](https://github.com/ManageIQ/manageiq-decorators/blob/master/app/assets/images/svg/vendor-power_vc.svg) isn't proportioned correctly (should be square) and is very hard to read. See this discussion: https://github.com/ManageIQ/manageiq-decorators/pull/49

**Comment by iv1111** [comment]
> @iv1111 I found some possibilities used for official PowerVC public accounts:

This would definitely look better but the question is whether we can use it as it's an outdated logo for PVC.

**Comment by jaywcarman** [comment]
I created a new icon updated in https://github.com/ManageIQ/manageiq-decorators/pull/50. I don't think it looks great, but it's better than what we've got and I think it complies with IBM branding requirements. fyi @jwcroppe

Here's a screenshot showing what it looks like from the provider overview page. For more screenshots see https://github.com/ManageIQ/manageiq-decorators/pull/50#issuecomment-881598493
![image](https://user-images.githubusercontent.com/1637291/125985334-32ab21c7-9c85-4f4a-89c9-0efea577a532.png)
